ext/digest/crc12_3gpp/crc12_3gpp.c

```diff
@@ -15,52 +15,42 @@
  */
```

Comment:
According to <https://github.com/tpircher/pycrc#copyright-of-the-generated-source-code>, the generated code can be treated as public domain.
Do we need any addenda as a digest-crc project to prevent misidentification?

Reply:
:black_cat: I can't see it because it's out of sight.

https://github.com/postmodern/digest-crc/blob/91ee75f6416cae5cbf0cd5a4c382ef71059dc951/ext/digest/crc12_3gpp/crc12_3gpp.c#L1-L15

Reply:
Probably not. We may need to change the wording to "Originally generated" or "Derived from", since I made some modifications by adding custom `typdef`s.

Reply:
:smiley_cat: I'll not do anything in this PR as I believe you will make this change if necessary.